ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/components/button/ButtonComponentView.kt

```diff
@@ -0,0 +1,91 @@
+package com.revenuecat.purchases.ui.revenuecatui.components.button
```

Comment:
Just in case, although not sure if needed for Composables: 

```suggestion
@file:JvmSynthetic

package com.revenuecat.purchases.ui.revenuecatui.components.button
```